malariagen_data/anopheles.py

```diff
@@ -6428,7 +6428,9 @@ def _haplotype_pairwise_distances(
             Hierarchically cluster haplotypes in region and produce an interactive plot.
         """,
         parameters=dict(
-            kwargs="Passed through to `px.scatter()`.",
+            leaf_y="Y coordinate at which to plot the leaf markers.",
```

Comment:
I decided to make a small visual change to the dendrogram to plot the leaves at Y coordinate zero by default, whereas previously it was -1. I did this because it creates a more visual connection between the dendrogram lines and the leaves. However, I made this a parameter so users can put the leaves back to -1 if they prefer.